custom-node/README.md

```diff
@@ -0,0 +1,100 @@
+# Custom Diem Node Starter Kit
+
+This folder contains what you need to start a Diem node locally with a custom genesis module set, along with
+some sample Move code and a sample application sending transactions to the local "blockchain" you 
+just started.
+
+## Folder Structure
+
+```
+.
+├── genesis                   # CLI for building and starting a validator node
+├── move/src                  # Move source code
+│   ├── diem                  # Diem Framework code
+│   ├── SampleModule.move     # A sample Move module
+├── sample-app                # A sample app talking to the blockchain
+├── transaction-builder       # Auto-generated transaction builders in Rust
+```
+## Step 0: Install Dependencies
+
+- Install Diem dependencies including Rust, Clang, etc, by running the following script in `diem` root directory:
+```
+./scripts/dev_setup.sh
+```
+- Install `df-cli` inside `diem` repo. We use this tool in step 2 to compile Move code:
+```
+cargo install --git https://github.com/diem/diem df-cli --branch main
+```
+
+## Step 1: Write Move Code
+
+Write some Move modules you wish to include in your custom genesis and put them under `./move/src/modules`, 
+such as the `SampleModule` we have included. This sample module contains a simple script function `mint_coin` that 
+publishes a resource called `Coin` as well as a unit test `test_mint_coin` that tests this function.
+
+Run `./move/publish.sh` to compile the Move code. No output means everything is good.
```

Comment:
This doesn't work - maybe should be changed to `cd move && bash publish.sh`

Reply:
Why doesn't that work? just make the script executable?

Reply:
This doesn't work because `publish.sh` has to be executed in the `move` folder. Will change the instructions.